action.yml

```diff
@@ -62,88 +62,39 @@ inputs:
+    value: ${{ steps.script.outputs.safesvg_count }}
   tfsec-count:
     description: number of tfsec findings via reviewdog
-    value: ${{ steps.run-reviewdog-pr.outputs.tfsec_count }}${{ steps.run-reviewdog-full.outputs.tfsec_count }}
+    value: ${{ steps.script.outputs.tfsec_count }}
   semgrep-count:
     description: number of semgrep findings via reviewdog
-    value: ${{ steps.run-reviewdog-pr.outputs.semgrep_count }}${{ steps.run-reviewdog-full.outputs.semgrep_count }}
+    value: ${{ steps.script.outputs.semgrep_count }}
   sveltegrep-count:
     description: number of sveltegrep findings via reviewdog
-    value: ${{ steps.run-reviewdog-pr.outputs.sveltegrep_count }}${{ steps.run-reviewdog-full.outputs.sveltegrep_count }}
+    value: ${{ steps.script.outputs.sveltegrep_count }}
   npm-audit-count:
     description: number of npm-audit findings via reviewdog
-    value: ${{ steps.run-reviewdog-pr.outputs.npm_audit_count }}${{ steps.run-reviewdog-full.outputs.npm_audit_count }}
+    value: ${{ steps.script.outputs.npm_audit_count }}
   pip-audit-count:
     description: number of pip-audit findings via reviewdog
-    value: ${{ steps.run-reviewdog-pr.outputs.pip_audit_count }}${{ steps.run-reviewdog-full.outputs.pip_audit_count }}
+    value: ${{ steps.script.outputs.pip_audit_count }}
 runs:
   using: 'composite'
   steps:
-    - name: Get changed files
-      if: ${{ inputs.enabled == 'true' && github.event_name == 'pull_request' }}
-      id: changed-files
-      uses: tj-actions/changed-files@0874344d6ebbaa00a27da73276ae7162fadcaf69 # v44.3.0
-      with:
-        separator: '\0'
-    - name: Store reviewdog enabled for PR
-      if: ${{ inputs.enabled == 'true' && inputs.baseline_scan_only != 'false' && steps.changed-files.outputs.all_changed_files != '' && github.event_name == 'pull_request' && github.event.pull_request.draft == false && github.actor != 'dependabot[bot]' }}
-      id: reviewdog-enabled-pr
-      shell: bash
-      run: |
-        set -x
-        echo "result=true" >> $GITHUB_OUTPUT
-    - name: Store reviewdog enabled for full repo manual run
-      if: ${{ inputs.enabled == 'true' && !(steps.reviewdog-enabled-pr.outputs.result == 'true') && (inputs.baseline_scan_only == 'false' || github.event_name == 'workflow_dispatch') }}
-      id: reviewdog-enabled-full
-      shell: bash
-      run: |
-        set -x
-        echo "result=true" >> $GITHUB_OUTPUT
     - name: Store reviewdog enabled
-      if: ${{ steps.reviewdog-enabled-pr.outputs.result == 'true' || steps.reviewdog-enabled-full.outputs.result == 'true' }}
+      # inputs.enabled == 'true' && (
+      #   (inputs.baseline_scan_only != 'false' && github.event_name == 'pull_request' && github.event.pull_request.draft == false && github.actor != 'dependabot[bot]') # reviewdog-enabled-pr
+      #   || 
+      #   (inputs.baseline_scan_only == 'false' || github.event_name == 'workflow_dispatch') # reviewdog-enabled-full
+      # )
```

Comment:
remove these commented-out lines?

Reply:
it has a second comment at the end, and it does not support multiline :S

I modify the commented part, then copy paste, and made it a oneliner